its/plugin/it-python-plugin-test/projects/test_code/tests/test_my_code.py

```diff
@@ -12,3 +12,10 @@ def test_something():
 class SomeCustomTestCase(MyTestClass):
         def test_addition(self):
             self.assertTrue(1 + 2 == 3)  # Issue here (S5906)
+
+
+from yest_another_test_file import MyOtherTestClass
```

Comment:
There is a small typo here for the imported module `yest` instead of `yet`

Reply:
Good catch!
It's unfortunately more than just a typo as the issue actually isn't found when the import is correctly made (and the class below uses the correct parent).
The initial goal of this test was to document this possible limitation, and I was surprised to see it succeed. This makes more sense now.